images/build/go-runner/cloudbuild.yaml

```diff
@@ -9,6 +9,12 @@ options:
   machineType: 'N1_HIGHCPU_8'
 
 steps:
+  - name: 'gcr.io/projectsigstore/cosign/ci/cosign@sha256:c581a4f0f6dd158220fa05d2351d8015f969b68de5c61d52c41478fae84e7064' # v0.3.1
```

Comment:
I said to Dan some time ago that we might want to start to publish the tagged image releases in another bucket that does not contain the `ci` on it to avoid confusion for the users.
But I know we all are working to make the cosign image a bit better before doing that. 😃

Reply:
Ah, it's published in both places actually! Good catch. The releases are at `gcr.io/projectsigstore/cosign:v0.3.1` as well.

Reply:
done